docs/ref-manual/cli/options.md

```diff
@@ -399,6 +399,53 @@ Where `TokenPair` is
 We have two contracts `BankToken.sol` and `LoanToken.sol`. We want `tokenA` of the `tokenPair` to be `BankToken`, and `tokenB` to be `LoanToken`. Addresses take up only one slot. We assume `tokenPair` is the first field of Bank (so it starts at slot zero). To do that, we use:  
 `certoraRun Bank.sol BankToken.sol LoanToken.sol --verify Bank:Bank.spec --structLink Bank:0=BankToken Bank:1=LoanToken`
 
+Options for controlling contract creation
+-----------------------------------------
+
+### `--dynamic_bound`
+
+**What does it do?**
+If set to zero (the default) dynamic creation will not be modeled. If non-zero, then dynamic contract creation will be modeled with cloning, where each contract will be cloned at most n times.
+**When to use it?**
+When you wish to model contract creation as a non-havoc.
+**Example**
+Suppose a contract `C` creates a new instance of a contract `Foo`, and you wish to inline the constructor of `Foo` at the creation site.
+`certoraRun C.sol Foo.sol --dynamic_bound 1`
+
+### `--dynamic_dispatch`
+
+**What does it do?**
+If false (the default), then all contract method invocations on newly created instances on spec must be modeled with explicit DISPATCHER summaries. 
+If true, then Prover will, on a best effort basis, automatically apply the DISPATCHER summary for call-sites that must be with a newly created contract as a receiver.
+**When to use it?**
+When you prefer not to add explicit `DISPATCHER` summaries to methods invoked by the created contract.
+**Example**
+Suppose a contract `C` creates a new instance of a contract `Foo`, and you wish to inline the constructor of `Foo` at the creation site, 
+and `Foo` calls some method `m()` which you wish to automatically link to the newly created contract.
+`certoraRun C.sol Foo.sol --dynamic_bound 1 --dynamic_dispatch true`
+
+### `--prototype`
+
+**What does it do?**
+Instructs the Prover to consider an hexadecimal string passed into a `create` or `create2` call as returning an instance of a specific contract.
```

Comment:
The string doesn't return anything.  I think "Instructs the Prover to use a specific contract type for the return value from a call to `create` or `create2` on the given hex string" is correct, but I'm not sure.